src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -364,14 +367,37 @@ private bool CanConnect(object parameter)
             return true;
         }
 
+        // Handler to invoke node Auto Complete
+        private void AutoComplete(object parameter)
+        {
+            // Since NodeAutoCompleteCommand is not triggered from xml event trigger directly,
+            // CanAutoComplete will not be triggered automatically.
+            if (CanAutoComplete(parameter))
+            {
+                DynamoViewModel dynamoViewModel = _node.DynamoViewModel;
+                var svm = dynamoViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel;
+                svm.PortViewModel = parameter as PortViewModel;
+                dynamoViewModel.CurrentSpaceViewModel.ShowNodeAutoCompleteSearchCommand.Execute(ShowHideFlags.Show);
```

Comment:
Why can't we make `OnRequestNodeAutoCompleteSearch` internal and call it directly from here instead of defining yet another command (`ShowNodeAutoCompleteSearchCommand`) here? Something like:
```
dynamoViewModel.CurrentSpaceViewModel.OnRequestNodeAutoCompleteSearch(ShowHideFlags.Show);
```

Reply:
I find too many "commands" is confusing.

Reply:
Will look into this

Reply:
@aparajit-pratap it works and I could remove the command `ShowNodeAutoCompleteSearchCommand` on workspaceViewModel layer. We may want to add recordable command support in the future and look at analytics.. But maybe not needed as a starting point?

Reply:
The reason why I thought this command was redundant is since we already have `NodeAutoCompleteCommand`, which I think in turn was calling this command, correct me if I'm wrong.

Reply:
@aparajit-pratap They are two different commands on different view model. But I have removed the one on workspace view model already.